.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+Not familiar with Git? Start by looking at GitHub's [collaborating pages](https://help.github.com/categories/collaborating/).
+
+The *pull request* process is important, especially the code submission part, KVIrc uses the automatic **travis** build system to compile the *pull request* and allows the person(s) involved the time to make any improvements or fix any issues arising before the **pr** is merged into the main source tree. 
+It also allows any interested **KVIrc development Team** members the opportunity to provide feedback and guidance where needed.
+
+###### General guidelines for creating pull requests:
+
+- **Create topic branches**. Don't ask us to pull from your master branch. 
+- **One pull request per feature**. If you want to do more than one thing, send multiple *pull requests*. 
+- **Send coherent history**. Make sure each individual commit message in your *pull
+  request* is meaningful.  
+  If you had to make multiple intermediate commits while developing, please squash them before sending them to us.  
+  In the end before merging you may be asked to squash your commit even some more.
+
+###### Try to follow these guidelines; it's the best way to get your work included in the project!
+
+* [**Click**](https://github.com/kvirc/KVIrc/fork/) to fork the **application** repo.
+* [**Click**](https://github.com/kvirc/kvirc-themes/fork/) to fork the **themes** repo.
+* [**Click**](https://github.com/kvirc/kvirc-scripts/fork/) to fork the **scripts** repo.
+* [**Click**](https://github.com/kvirc/kvirc-www/fork/) to fork the **website** repo.
+* [**Configure**](https://help.github.com/articles/configuring-a-remote-for-a-fork/) the remote.
+
+***
+
+Clone your fork of your chosen project repo you did above into the current directory in terminal.
+   ```bash
+   # the example below uses the main repo, please adapt it to match the repo you have forked.
+   $ git clone git@github.com:<your GitHub username>/xbmc.git my-kvirc
+   # Navigate to the newly cloned directory
+   $ cd my-kvirc 
+   # Assign the original repo to a remote called e.g. "upstream"
+   $ git remote add upstream https://github.com/kvirc/KVIrc.git
+   ```
+
+If you cloned/forked a while ago, get the latest changes from upstream:
+
+   ```bash
+   # Make sure you are on your 'master' branch
+   $ git checkout master
+   # Fetch upstream changes and rebase
+   $ git pull --rebase upstream master
+   # Push those changes to GitHub
+   $ git push -f
+   ```
+
+Create a new topic branch to contain your feature, change, or fix:
+
+   ```bash
+   $ git checkout -b <topic-branch-name>
+   ```
+
+Commit your changes in logical chunks, or your *pull request* is unlikely to be merged into the main project.  

```

Comment:
You could also add that if the commit fixes a bug, they can automatically close the ticket by adding `Closes #ticketnumber` somewhere in the commit message.